### ISSUE TITLE
Clean up LP Design system on Figma

### ISSUE THREAD

**Issue by sandyklc** [ubclaunchpad/design#22]
Fix design files and design system on Figma.

**Comment by bobheadxi** [comment]
Would be good to link the "main" Launch Pad branding Figma here and in the handbook! I think it should be okay to make that one read-only for everyone with link

**Comment by sandyklc** [comment]
Yup definitely, I will link it once I finish cleaning it up. I'm probably going to make a new project so that people don't have to go through the old figma project (Since its quite disorganized and hard to find stuff)

**Comment by bobheadxi** [comment]
From #17 :

> in instances where we have two accounts (for example, a Launch Pad group and Launch Pad user), use the light logo for the group and the dark logo for the user. Otherwise use the dark logo
